lms/envs/devstack.py

```diff
@@ -230,6 +230,14 @@ def should_show_debug_toolbar(_):
 if os.path.isfile(join(dirname(abspath(__file__)), 'private.py')):
     from .private import *  # pylint: disable=import-error,wildcard-import
 
+# do this after private.py, ensuring this section picks up credential overrides.
```

Comment:
This comment doesn't make much sense. Also, by updating this after `private.py` is loaded, you prevent `private.py` from being able to override the settings. This block should come *before* `private.py`.